src/UsersData.tsx

```diff
@@ -0,0 +1,23 @@
+export interface User {
+  id: number;
+  name: string;
+  surname: string;
+  username: string;
+  password: string;
+  email: string;
+  profilePicture?: string | null;
+}
+
+export const allUsers: User[] = [
```

Comment:
From the user facing UI perspective, there will always be a single user in the application. It's in the backend the application can support multiple users.

For now, change it to be just a `user` (instead of `allUsers`). You'll need to update `users` property and `setUsers` method in AppContext as well to only support a single user.